Description: Don't run dynamic_lib::tests::test_loading_cosine on Aarch64
 whilst it fails.
Author: Chris Coulson <chris.coulson@canonical.com>
Bug: https://github.com/rust-lang/rust/issues/45410

--- a/src/librustc_metadata/dynamic_lib.rs
+++ b/src/librustc_metadata/dynamic_lib.rs
@@ -113,7 +113,7 @@
 
     #[test]
     fn test_loading_cosine() {
-        if cfg!(windows) {
+        if cfg!(any(windows, target_arch = "aarch64")) {
             return
         }
 
